sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -247,6 +247,12 @@ public EventDirectorySteps(
           TimeUnit.SECONDS.sleep(10);
         });
 
+    When(
+        "I click on the created event participant from the list",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(CREATED_PARTICIPANT);
```

Comment:
this is not always the best approach, because if another test will run in parallel, the first result can be from other test.
Update the locator to indetify the participant based on partial ID, so here the uuid can be collected from created entity and formatted to generate the webelement and click

Reply:
We're doing exactly that searching for the event we've just created based on its ID, and only then we create and open the event participant from inside that specific event (not from the general list) - so there's no chance that another test interferes, as it would have to purposefully search for the very same event first in order to create another participant inside it.